src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -177,6 +178,11 @@ public String getQuestionResultStatisticsHtml(
 
         Map<String, List<Integer>> recipientRanks = generateOptionRanksMapping(responses);
 
+        boolean isAvgExclSelfShown = shouldSelfBeExcludedFromRankings(question);
```

Comment:
If you can collapse everything into a single `if` statement defined in a separate method, this variable won't even be necessary. While variable abstraction is good in general, overuse makes the code harder to follow.